IPython/core/inputsplitter.py

```diff
@@ -687,7 +686,6 @@ def push(self, lines):
         """
 
         # We must ensure all input is pure unicode
-        lines = cast_unicode(lines, self.encoding)
```

Comment:
Why is this changed in a doc PR? If you're sure removing it is OK, please remove the comment too.